sample_project/sample_project/app/v1/books/service.py

```diff
@@ -0,0 +1,92 @@
+from flask import jsonify
+
+from app.connections import SqlConnection
+class SubApp1Service:
+    db_config = {
+        'host': 'localhost',
```

Comment:
This credentials should come from config and not in code.

Reply:
ok I will make changes